src/Symfony/Component/Console/Command/InvokableCommand.php

```diff
@@ -30,26 +30,28 @@
  */
 class InvokableCommand implements SignalableCommandInterface
 {
-    private readonly \Closure $code;
+    private readonly \Closure $closure;
     private readonly ?SignalableCommandInterface $signalableCommand;
     private readonly \ReflectionFunction $reflection;
     private bool $triggerDeprecations = false;
+    private $code;
 
     public function __construct(
         private readonly Command $command,
         callable $code,
```

Comment:
```suggestion
        private callable $code,
```

Reply:
When we make this change, PHP gives an error: "Property cannot have type 'callable'". See https://stackoverflow.com/questions/57935734/is-type-callable-supported-with-typed-properties.

Reply:
The `callable` pseudo-type cannot be used as property type. That's why CPP isn't used here.

https://github.com/symfony/symfony/pull/61078#discussion_r2197469426